CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ else (WIN32)
       # Ensure that objcopy is present
       if (DEFINED ENV{CROSSCOMPILE} AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
         if (CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l OR CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64)
-          find_program(OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
+          find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
```

Comment:
To cross-compile for Android, we need to use objcopy from the Android toolchain, not the host. On Android, that would be `${CROSS_NDK_TOOLCHAIN}/bin/aarch64-linux-android-objcopy`; previously we set `${TOOLCHAIN_PREFIX}` to `${CROSS_NDK_TOOLCHAIN}/bin/aarch64-linux-android-` so it worked.

If you set it to `${TOOLCHAIN}` this will evaluate to `aarch64-linux-android-objcopy` and from what I get, CMake will look for this file in the bin folders on the host, which doesn't work.

Another option would be to set `${TOOLCHAIN_PREFIX}` to `${CROSS_NDK_TOOLCHAIN}/bin/` on Android, leave it empty for Linux, and change this line to:

```
 find_program(OBJCOPY ${TOOLCHAIN_PREFIX}${TOOLCHAIN}-objcopy)
```